backend/strapi-sync/sync.js

```diff
@@ -0,0 +1,404 @@
+import { get } from "./axios.js";
+import {
+  getItemByAttributes,
+  createModel,
+  getStrapiModelData,
+} from "./utils.js";
+import {
+  Park,
+  FeatureType,
+  Feature,
+  Dateable,
+  DateType,
+  Season,
+  DateRange,
+} from "../models/index.js";
+import { Op } from "sequelize";
+
+async function getPageData(url) {
+  try {
+    const response = await get(url);
+    const data = await response.json();
+
+    return data.data;
+  } catch (error) {
+    console.error(error);
+
+    return error;
+  }
+}
+
+export async function getData(url, queryParams) {
+  const items = [];
+
+  try {
+    let currentUrl = url;
+
+    if (queryParams) {
+      currentUrl = `${currentUrl}?${queryParams}`;
+    }
+
+    const response = await get(currentUrl);
+    const data = await response.json();
+    const pagination = data.meta.pagination;
+
+    items.push(...data.data);
+
+    let page = 1;
+
+    while (page < pagination.pageCount) {
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+      page += 1;
+
```

Comment:
We could use a `for` loop to save having to manually increment `page`:
```suggestion
    for (let page = 1; page < pagination.pageCount; page++) {
      await new Promise((resolve) => setTimeout(resolve, 1000));

```

I'm guessing that 1s pause is to avoid overloading the API? Let's throw some comments in to say why it's doing that